src/ralph/api/routers/statements.py

```diff
@@ -435,8 +446,8 @@ async def put(
 
     # For valid requests, perform the bulk indexing of all incoming statements
     try:
-        success_count = BACKEND_CLIENT.write(
-            data=[statement_as_dict], ignore_errors=False
+        success_count = await await_if_coroutine(
```

Comment:
Maybe its name should be changed for more readability (avoid `await` duplication)

Reply:
The duplication is really here! We await the wrapper -> we await if it's a coroutine else we return

Reply:
okay !